recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "rubin-env" %}
-{% set version = "0.2.1" %}
+{% set version = "0.3.1" %}
```

Comment:
If bumping minor version, can you start at 0.3.0?

Reply:
Of course.